test/sequential/test-timers-block-eventloop.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const common = require('../common');
+const fs = require('fs');
+
+const t1 = setInterval(() => {
+  common.busyLoop(12);
+}, 10);
+
+const t2 = setInterval(() => {
+  common.busyLoop(15);
+}, 10);
+
+const t3 = setTimeout(() => {
+  throw new Error('eventloop blocked!');
```

Comment:
Why throw here? If the intent is that this function should not be called, then using `common.mustNotCall()` would be better.

Reply:
Yes, it's better, updated as your suggestion.